src/pages/MuteerBeleidsmodulesOverview/TableRow/TableRow.tsx

```diff
@@ -0,0 +1,58 @@
+import { Link } from 'react-router-dom'
+
+import {
+    BeleidsmodulesReadMaatregelenItem,
+    BeleidsmodulesReadBeleidskeuzesItem,
+} from '@/api/fetchers.schemas'
+
+import * as BELEIDSKEUZES from '../../../constants/beleidskeuzes'
```

Comment:
@Aidenbuis You can use the new way of importing file like `import * from BELEIDSKEUZES from '@/constants/beleidskeuzes'`

`@/*` is set to `./src/*`